lib/openstax/aws/stack.rb

```diff
@@ -297,7 +297,16 @@ def self.format_hash_as_stack_parameters(params={})
     end
 
     def status
-      aws_stack.stack_status
+      begin
+        aws_stack.stack_status
+      rescue Aws::CloudFormation::Errors::ValidationError => ee
+        case ee.message
+        when /Stack.*does not exist/
+          "DOES_NOT_EXIST"
+        else
```

Comment:
"DOES_NOT_EXIST" as a class constant? (then we can have just one copy of this string, referred to in two different places, and potentially more down the road.)